pkg/oomstore/snapshot.go

```diff
@@ -30,7 +30,7 @@ func (s *OomStore) Snapshot(ctx context.Context, groupName string) error {
 		}
 	}
 	for i, revision := range revisions {
```

Comment:
after creating first snapshot, we need to re-fetch revisions to get the latest data

Reply:
re-fetch informer?